control_plane/src/local_env.rs

```diff
@@ -57,6 +58,11 @@ pub struct LocalEnv {
 
     #[serde(default)]
     pub safekeepers: Vec<SafekeeperConf>,
+
+    /// Every tenant has a first timeline created for it, currently the only one ancestor-less for this tenant.
+    /// It is used as a default timeline for branching, if no ancestor timeline is specified.
```

Comment:
The CLI `zenith branch' command requires you to specify the parent branch. Is there any scenario where the ancestor isn't given explicitly?